### ISSUE TITLE
Allow `command` entry in Podman RuntimeConfig to overrides image's entrypoint

### ISSUE THREAD

**Issue by windsource** [eclipse-ankaios/ankaios#24]
Reported by @christoph-hamm:

## Description

The `command` entry in the Podman RuntimeConfig shall override the `entrypoint` of the Podman image. At the moment `command` entry is combined with `args` entry and used as `command` argument when creating the Podman container.

The naming here is little bit confusing as `command` means something different in the RuntimeConfig, than in the Podman API. The terms `command` and `args`, as used in the RuntimeConfig, have been taken from the Kubernetes Containers definition.

## Goals

It is possible to override the `Entrypoint` of a Podman image, using the `command` entry in the RuntimeConfig.

## Tasks

- [x] Move Podman-related functions (i.e. functions using Podman API) to the `podman_utils.rs`. PR:  #40 
- [x] Forward the value of `command` entry of the RuntimeConfig to the `entrypoint` parameter when creating a Podman container - #45 
- [x] Keep forward the value of `args` entry of the RuntimeConfig to the `command` parameter when creating a Podman container - #45 
- [x] Fix the unit tests - unit tests will be created in #1 
- [x] Checks if it is now possible to override the Podman image's `Entrypoint` - see info below

**Comment by maturar** [comment]
This ticket is a chance for small refactoring. We would like to move all functions which call Podman API from the `workload.rs` to the `podman_utils.rs`. The motivation is to have podman-independent code in the `workload.rs`. I have created a task for that.

**Comment by maturar** [comment]
**Regarding testing**
We have agreed that we are not going to implement unit tests in this ticket. Current unit tests do not cover how the runtime configuration is mapped into Podman. The mock (Podman test daemon) does not cover entrypoint of the container. Additionally we are going to do massive changes in this area, which will include integration tests. See #1 and #19 . 

I have tested the change manually. I have created own docker image (called "testprintenv") from this ```Dockerfile```:
```
FROM ubuntu:latest
ENTRYPOINT env
```
The point is that this image contains the entrypoint and with the fix from this ticket the Ankaios shall be able to override the entrypoint.
I have created two configurations (startConfig.yaml) with two different runtime configs:

1. 
``` 
    runtimeConfig: |
      image: localhost:5000/testprintenv:latest
```
2. 
```
    runtimeConfig: |
      image: localhost:5000/testprintenv:latest
      command: ["echo"]
      args: ["Hello Ankaios"]
```
The correct behavior is if I run the first configuration the container shall print the environment variables. The second container shall print the message "Hello Ankaios". With the fix this happens. With current main both configurations print the environment variables (which is not okay).

One more thing I had to change here. The ```command``` and ```args``` in the runtime configuration in the Ankaios shall be optional. We map the ```command``` to ```entrypoint``` in the container. Podman and docker say are able to create a container which uses the ```entrypoint``` from the image. We have to be able to do the same in the Ankaios.
